data/reusables/dependabot/supported-package-managers.md

```diff
@@ -116,9 +117,11 @@ For {% data variables.product.prodname_dependabot_security_updates %}, Gradle su
 
 {% ifversion dependabot-PEP621-support %}
 
-#### pip and pip-compile
+#### pip-compile and uv
 
-In addition to supporting updates to `requirements.txt` files, {% data variables.product.prodname_dependabot %} supports updates to `pyproject.toml` files if they follow the PEP 621 standard.
+{% data variables.product.prodname_dependabot %} can update `requirements.txt` compiled files using both `pip-compile` and `uv pip` (depending on which one was used to generate the file), as well as performing updates to `pyproject.toml` files if they follow the PEP 621 standard.
```

Comment:
```suggestion
{% data variables.product.prodname_dependabot %} can update `requirements.txt` files using both `pip-compile` and `uv pip` (depending on which one was used to generate the file). {% data variables.product.prodname_dependabot %} can also update `pyproject.toml` files if they follow the PEP 621 standard.
```

This is just to try to make the language a bit simpler and avoid the slightly awkward part "update `requirements.txt` compiled files"

Reply:
Separately, in this context, I think saying `uv` might make more sense? ie "... using both `pip-compile` and `uv`... "

I haven't raised it as a suggestion.

Reply:
Haha I hadn't seen that—I won't argue with the person writing the tool! I think let's go with the first suggestion in that case 😄